Configuration/Settings.yaml

```diff
@@ -45,6 +45,8 @@ Neos:
 Sandstorm:
   NeosTwoFactorAuthentication:
     # enforce 2FA for all users
-    enforceTwoFactorAuthentication: false
+    enforceTwoFactorAuthenticationForAllUserAccounts: false
```

Comment:
I'm in favor of expressive variable names, but maybe we can reduce noise here by dropping the `TwoFactorAuthentication` part.

```diff
-- enforceTwoFactorAuthenticationForAllUserAccounts: false
-- enforceTwoFactorAuthenticationForAuthenticationProviders : []
-- enforceTwoFactorAuthenticationForSpecificUserAccounts: []

++ enforceForAllUserAccounts: false
++ enforceForAuthenticationProviders : []
++ enforceForSpecificUserAccounts: []
```

Not necessary and just an idea.

Reply:
What about `enforce2FAFor...`?

Reply:
I also documented the settings wihtin the README.

Thanks for your comments and this awesome package.

Reply:
Replaced option to verify roles instead of specific users